utilities/provider_tallies/README.md

```diff
@@ -0,0 +1,16 @@
+# Provider Tally script
+
+This script is used to extract the per-provider counts for both total results
+and appearances within searches from the Openverse API redis.
+
+## Usage
+
+1. `pipenv install`
+2. `pipenv run python provider_tally_stats.py`
+
```

Comment:
These are the instructions for testing with the API locally, right? Doesn't it need to pass Redis credentials anywhere?

Reply:
Ah, there's a note about this on the script itself, but not in the README: https://github.com/WordPress/openverse/pull/397/files#diff-fb0b292bf6766acd757c8b72104edc2d46a78ab4e0d5738bb86e4d875965f91aR4-R6

I'll add it there too 🙂